pyproject.toml

```diff
@@ -5,6 +5,19 @@ description = "Honeycomb OpenTelemetry Distro for Python"
 authors = ["Honeycomb <support@honeycomb.io>"]
 readme = "README.md"
 packages = [{include = "honeycomb", from = "src" }]
+license = "Apache-2.0"
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: Apache Software License",
```

Comment:
👍🏻 I wish this was "Apache License, Version 2.0" but that's not the string the ecosystem appears to expect.